conf/script/build-ha-csm

```diff
@@ -115,6 +119,11 @@ fi
     exit 1
 }
 
+[[ $cidr ]] || {
+    # setup cidr netmask by default
```

Comment:
You can assign default value at line 79 and drop this whole block.

Reply:
Updated pull request.